### DISCUSSION TITLE
How to get form last step value in Bindings?

### DISCUSSION THREAD

**Discussion by arladmin** [Budibase/budibase#6684] [Help]
I would like to make changes to a component based on whether the form step is the last one or not.

Is there any way to check for this, just like we can check for ```{{ Form.Current Step }} == 1``` ?

**Comment by proton029** [comment]
Hi there. Well one way  is to add a configuration for the component
Here i have added a placeholder for the long text component when it reaches the currentstep
![image](https://user-images.githubusercontent.com/88423620/178923526-6f047a6b-ef04-498c-b094-48d6b7c3d761.png)
